test/MicrosoftTeams.spec.ts

```diff
@@ -1138,6 +1138,46 @@ describe("MicrosoftTeams", () => {
       expect(message.args[2]).toBe(authenticationParams.height);
     });
 
+    it("should successfully pop up the auth window in the android client", () => {
```

Comment:
NIT: You can implement the android, ios, and desktop test cases using a single code block inside of a loop, i.e.:
```
["android", "ios", "desktop].forEach(hostClientType => {
  it(`should successfully handle auth success in the ${hostClientType} client`, () => {
    initializeWithContext("content", hostClientType);
    ...
  });
});
```

Reply:
Nice !

Reply:
Done.
Should i also do it for the auth success and auth failure cases?

Reply:
I've added them for auth success and failure, since the tests do fail if the client types are removed